3-networks/envs/shared/hierarchical_firewall.tf

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "hierarchical_firewall_policy" {
+  source = "../../modules/hierarchical_firewall_policy/"
+  parent = data.google_active_folder.common.name
+  name   = "common-firewall-rules"
+  associations = [
+    data.google_active_folder.common.name,
+    data.google_active_folder.bootstrap.name,
+    data.google_active_folder.development.name,
+    data.google_active_folder.production.name,
+    data.google_active_folder.non-production.name,
+  ]
+  rules = {
+    delegate-rfc1918-ingress = {
+      description = "Delegate RFC1918 ingress"
+      direction   = "INGRESS"
+      action      = "goto_next"
+      priority    = 500
+      ranges = [
+        "192.168.0.0/16",
+        "10.0.0.0/8",
+        "172.16.0.0/12"
+      ]
+      ports                   = { "all" = [] }
+      target_service_accounts = null
+      target_resources        = null
+      logging                 = false
+    }
+    delegate-rfc1918-egress = {
+      description = "Delegate RFC1918 egress"
+      direction   = "EGRESS"
+      action      = "goto_next"
+      priority    = 510
+      ranges = [
+        "192.168.0.0/16",
+        "10.0.0.0/8",
+        "172.16.0.0/12"
+      ]
+      ports                   = { "all" = [] }
+      target_service_accounts = null
+      target_resources        = null
+      logging                 = false
+    }
+    allow-iap-ssh-rdp = {
+      description             = "Always allow SSH and RDP from IAP"
```

Comment:
In the previous configuration, these rules were optional and disabled by default. https://github.com/terraform-google-modules/terraform-example-foundation/blob/develop/3-networks/envs/development/variables.tf#L107

Are we changing that behaviour here?

Reply:
Indeed. Before, SSH/RDP from IAP was optional, whereas now is always enabled.

Reply:
Ok cool - just clarifying that is intended with the team as in the previous release we explicitly did the opposite.